sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/SaveIntoDataSourceCommand.scala

```diff
@@ -46,6 +46,10 @@ case class SaveIntoDataSourceCommand(
   override def innerChildren: Seq[QueryPlan[_]] = Seq(query)
 
   override def run(sparkSession: SparkSession): Seq[Row] = {
+    if (options.values.exists(_ == null)) {
```

Comment:
How about `keys`? Should we check them too?

Reply:
No, the keys cannot be null since Map[String, String] can't accept null keys.